### PR TITLE
Starting tasks support

### PR DESCRIPTION
**Pull request by enxebre**
This introduce very basic support for tasks
Follow ups:
https://github.com/Capgemini/mesos-ui/issues/76
https://github.com/Capgemini/mesos-ui/issues/77
https://github.com/Capgemini/mesos-ui/issues/75

Table
screen shot 2016-03-08 at 17 41 13
Visualizer
screen shot 2016-03-08 at 17 40 50

### CODE REVIEW

**Comment by wallies** [comment]
these codeclimate build failures are fixed by #79

File: src/components/TaskVisualizer/TaskVisualizer.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
+/*jshint esnext: true */
+
+import React, { PropTypes } from 'react';
+import DashboardBox from '../../components/DashboardBox';
+import Sunburst from '../../components/Sunburst';
+import Circle from '../../components/Circle';
+import random from 'lodash/number/random';
+import range from 'lodash/utility/range';
+import List from 'material-ui/lib/lists/list';
```
Comment (tayzlor) [inline src/components/TaskVisualizer/TaskVisualizer.js]: v minor but lines like these can be condensed to ```import { List, ListItem } from 'material-ui/lib/lists```

File: src/components/TaskVisualizer/TaskVisualizer.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+  constructor() {
+    super();
+    this.state = {
+      parameterToGroupBy: 'name',
+      toggleAppList: {}
+    };
+  }
+
+  stringToColour(str) {
```
Comment (tayzlor) [inline src/components/TaskVisualizer/TaskVisualizer.js]: whats this function all about?

File: src/components/TaskVisualizer/TaskVisualizer.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+      }
+    }
+    return tasks;
+  }
+
+  createCirclesInGroupsList() {
+
+    let tasks = this.excludeToggledTasks();
+    let appColorList = this.createAppColorList();
+    let circlesList = {}
```
Comment (tayzlor) [inline src/components/TaskVisualizer/TaskVisualizer.js]: some end of lines have semi-colons others dont

**Comment by tayzlor** [comment]
:tada:

**Pull request merged**